### PR TITLE
Write tests for extract_gog.py

### LINKED ISSUE #49: Test extract_gog.py
## Description
Write a test file to test extract_gog.py

## Required Files
- test_extract_gog.py

## User Story
As an engineer I need well tested scripts so I am more confident they won't break.

### PR DESCRIPTION
**Pull request by annalisev**
Tested the small functions which don't rely on scraping. The functions which do rely on scraping are tested by running the script itself.

Closes #49 .

### CODE REVIEW

File: pipeline/extract/.coverage
Comment (shindym) [inline pipeline/extract/.coverage:1]: Does this need commiting?

File: pipeline/extract/conftest.py [hunk 1]
```diff
@@ -0,0 +1,47 @@
+import pytest
+
+from bs4 import BeautifulSoup
+
+
+@pytest.fixture
```
Comment (shindym) [inline pipeline/extract/conftest.py:6]: Good use of conftest.py

File: pipeline/extract/.coverage
Comment (annalisev) [reply pipeline/extract/.coverage:1]: No sorry!

File: pipeline/extract/extract_gog.py [hunk 2]
```diff
@@ -40,22 +41,28 @@ def get_detail_links(game_soup: BeautifulSoup) -> list:
         'a', class_='details__link')
 
 
-def get_json(game_soup: BeautifulSoup) -> dict:
+def get_game_data_json(game_soup: BeautifulSoup) -> dict:
```
Comment (shindym) [inline pipeline/extract/extract_gog.py:44]: Better function naming :)

File: pipeline/extract/extract_gog.py [hunk 3]
```diff
@@ -40,22 +41,28 @@ def get_detail_links(game_soup: BeautifulSoup) -> list:
...
     '''Returns a JSON object about a given game soup.'''
     return json.loads(game_soup.find(
         'script', type='application/ld+json').text)
 
 
-def get_developer(links: list) -> str:
+def get_developer(links: list[BeautifulSoup]) -> str:
```
Comment (shindym) [inline pipeline/extract/extract_gog.py:50]: Good use of typehints

**Review by shindym — APPROVED** [review]

**Review by Eibironke — APPROVED** [review]
👍

**Pull request merged**